fs/operations/check.go

```diff
@@ -56,6 +56,7 @@ type checkMarch struct {
 	dstFilesMissing int32
 	matches         int32
 	opt             CheckOpt
+	progress        bool
```

Comment:
> Maybe we should put this code in `operations.syncPrintf` (and make it public and rename the current SyncPrintf to something more sensible)?

Then `SyncPrintf` will have to check its file descriptor. If it's `os.Stdout`, it will use the progress hook (if any). If not, it will output to the file like before. Right?

Reply:
Implemented by https://github.com/rclone/rclone/pull/5352/commits/45196536ba46a246a84bf56ba85094f5b068a7de
PTAL :pray: